src/components/gcds-site-menu/gcds-site-menu.css

```diff
@@ -345,69 +373,147 @@
   margin-top: 0;
   text-align: left;
 }
-:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [role=menuitem] {
-  color: #FFFFFF;
+:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [role='menuitem'] {
+  color: #000000;
 }
 :host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [data-h2-submenu-trigger] [data-h2-submenu-trigger-add-icon],
 :host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [data-h2-submenu-trigger] [data-h2-submenu-trigger-remove-icon] {
-  color: #FFFFFF;
+  color: #000000;
 }
-:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li:not(:first-child) > [role=menuitem] {
-  border-top: 1px solid #613DC1;
+:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li:not(:first-child) > [role='menuitem'] {
+  border-top: 1px solid rgba(0, 0, 0, 0.3);
 }
-:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] + [data-h2-menulist] > li > [role=menuitem] {
-  border-top: 1px solid #613DC1;
+:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] + [data-h2-menulist] > li > [role='menuitem'] {
+  border-top: 1px solid rgba(0, 0, 0, 0.3);
 }
 :host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > a:hover {
   color: #8ffff3;
 }
 :host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [data-h2-submenu-trigger] {
-  background-color: rgba(255, 255, 255, 0.1);
+  background-color: rgba(130, 130, 130, 0.35);
+  transition: all 0.2s ease-in-out;
+}
+:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [data-h2-submenu-trigger]:hover {
+  background-color: rgba(130, 130, 130, 0.7);
+}
+:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [data-h2-submenu-trigger]:focus {
+  background-color: #303fc3;
+  color: #ffffff !important;
+  outline: none;
+}
+:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [data-h2-submenu-trigger]:focus span {
+  color: #ffffff !important;
+  transition: all 0.4s ease-in-out;
+}
+:host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [data-h2-submenu-trigger]:active {
+  background-color: #595959;
 }
 :host([data-h2-menu-wrapper]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] > li > [data-h2-menulist] {
-  background-color: #282832;
+  background-color: rgba(210, 210, 210, 0.5);
```

Comment:
When the looking at sub sub menu, so second level dropdown menu. The text on the page can make the bottom menu item hard to read. Maybe a slight increase in the background opacity can help with this.